src/args/default.nix

```diff
@@ -62,6 +62,13 @@ let
     sortCaseless = builtins.sort
       (a: b: with __nixpkgs__.lib.strings; toLower a < toLower b);
 
+    # Dumps an expression to a file in JSON format
+    toJSONFile = name: expr: args.makeDerivation {
+      env.envAll = builtins.toJSON expr;
+      builder = ''echo "$envAll" > $out'';
+      inherit name;
+    };
+
```

Comment:
for the record, the difference between this and `builtins.toFile name (builtins.toJSON expr)` is that this one can contain references to the nix store, while the builtin cannot